osi_lane.proto

```diff
@@ -678,6 +691,122 @@ message Lane
+        // at parts of the lane where height does not change much.
+        //
+        // \note The surfaceline is the line that describes the height profile 
+        // along the lane.
+        // 
+        // \attention If more than one surfaceline exists, they are not allowed to cross each other 
+        // or the lane boundaries, otherwise you could have conflicting height information at these points.
+        //
+        // \proposal In order to model curb stones, one surfaceline will be placed at the lower edge
+        // of the curb that belongs to the border lane/driving lane and another one on the upper edge
+        // that will belong to the sidewalk 
+        // (see https://github.com/OpenSimulationInterface/open-simulation-interface/pull/608)
+        //
+        //
+        message SurfaceLine
+        {
+            // The ID of the surfaceline.
+            // Example: sl 3 (see \ref Surfacelines).
+            //
+            // \note Note ID is global unique.
+            //
+            // \rules
+            // is_globally_unique
+            // \endrules
+            //
+            optional Identifier surfaceline_id = 1;
+    
+            // points making up the surfaceline
+            //
+            // If the surfaceline approximates a height curve, the points must be chosen 
+            // in a way that the lateral distance to the ideal line does not exceed 2cm. 
+            // As shown in the following image:
+            //
+            // \image html line_approximation_error.svg "Approximation error"
+            // Approximation error green line.
+            //
+            // This only applies to the approximation in height in comparison to the 
+            // surrounding flat surface plane.
+            //
+            // The "first" point of a surfaceline does not necessarily have to be at the 
+            // start of the lane it describes if the height does not change along its course.
+            // For the two surfaceline at the LaneBoundary it does make sense to start
+            // at the very beginning of that LaneBoundary in order to be able to describe the
+            // height plane of that Lane accordingly.
+            //
+            repeated SurfaceLinePoint surfaceline = 2;
+       
+            // Definition of the placement of the newly introduced surfaceline in the center of the lane.
+            //
+            // Defined and used for driving lanes.
+            // \c true means that the surfaceline is also the centerline of the lane.
+            //
+            // Example: \c #surfaceline_is_centerline = \c true for surfaceline sl3 
+            // and \c #surfaceline_is_centerline = \c false for surfacelane sl6 in
+            // image \ref Surfacelines .
+            // 
+            // \note Centerline should be deprecated in the near future in favor of 
+            // the logical lanes for driving and navigation purposes
+            //
```

Comment:
I think I would drop this because you already mentioned it in the deprecation comment of the centerline.
```suggestion
```